administrator/modules/mod_logged/tmpl/default.php

```diff
@@ -54,7 +54,7 @@
 					<?php endif; ?>
 				</td>
 				<td>
-					<?php echo HTMLHelper::_('date', $user->time, Text::_('DATE_FORMAT_LC5')); ?>
+					<?php echo HTMLHelper::_('date.relative', JFactory::getDate($user->time)); ?>
```

Comment:
```suggestion
					<?php echo HTMLHelper::_('date.relative', Factory::getDate($user->time)); ?>
```
Please add `use Joomla\CMS\Factory;`

Reply:
Doh, Im context switching so much at the moment Im forgetting which version of Joomla or symphony Im meant to be working in :)